config/puma.rb

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env puma
+
+stdout_redirect '/cellect_panoptes/log/puma_stdout.log', '/cellect_panoptes/log/puma_stderr.log'
```

Comment:
I saw that option, but not sure which is the better choice. I would've said append is less resource intensive, but I've got to think there's a reason it's not the default. I'd say let's keep it off for now, unless we find a compelling reason to enable it.